spaceflights/prompts.yml

```diff
@@ -0,0 +1,29 @@
+project_name:
+  title: "Project Name:"
+  text: |
+    Please enter a human readable name for your new project.
+    Spaces and punctuation are allowed.
+  default: New Kedro Project
```

Comment:
How are defaults handled for the prompts below? I know atm the default does not appear in the prompts (`[new_kedro_project]: `), is that something we can change as part of this change or not really?

Reply:
At the moment a default appears in the prompt if it is specified in `prompts.yml` but currently we do not support the behavior when `project_name` was used as a default for other inputs. Now you need to type all the inputs (if no defaults).